src/util/color/predefinedcolorpalettes.cpp

```diff
@@ -0,0 +1,160 @@
+#include "predefinedcolorpalettes.h"
+
+namespace {
+
+constexpr mixxx::RgbColor kColorMixxxRed(0xC50A08);
+constexpr mixxx::RgbColor kColorMixxxYellow(0x32BE44);
+constexpr mixxx::RgbColor kColorMixxxGreen(0x0044FF);
+constexpr mixxx::RgbColor kColorMixxxCeleste(0xF8D200);
+constexpr mixxx::RgbColor kColorMixxxBlue(0x42D4F4);
+constexpr mixxx::RgbColor kColorMixxxPurple(0xAF00CC);
+constexpr mixxx::RgbColor kColorMixxxPink(0xFCA6D7);
+constexpr mixxx::RgbColor kColorMixxxWhite(0xF2F2FF);
+
+constexpr mixxx::RgbColor kRekordboxTrackColorPink(0xF870F8);
+constexpr mixxx::RgbColor kRekordboxTrackColorRed(0xF870900);
+constexpr mixxx::RgbColor kRekordboxTrackColorOrange(0xF8A030);
+constexpr mixxx::RgbColor kRekordboxTrackColorYellow(0xF8E331);
+constexpr mixxx::RgbColor kRekordboxTrackColorGreen(0x1EE000);
+constexpr mixxx::RgbColor kRekordboxTrackColorAqua(0x16C0F8);
+constexpr mixxx::RgbColor kRekordboxTrackColorBlue(0x0150F8);
+constexpr mixxx::RgbColor kRekordboxTrackColorPurple(0x9808F8);
+
+constexpr mixxx::RgbColor kTraktorProTrackColorRed(0xFA4B35);
+constexpr mixxx::RgbColor kTraktorProTrackColorOrange(0xFF8402);
+constexpr mixxx::RgbColor kTraktorProTrackColorYellow(0xFFF700);
+constexpr mixxx::RgbColor kTraktorProTrackColorGreen(0x00F329);
+constexpr mixxx::RgbColor kTraktorProTrackColorBlue(0x0187FF);
+constexpr mixxx::RgbColor kTraktorProTrackColorViolet(0xA669FF);
+constexpr mixxx::RgbColor kTraktorProTrackColorMagenta(0xFE55EA);
+
+constexpr mixxx::RgbColor kSeratoDJProTrackColorGrey1(0x333333);
```

Comment:
Are these the visual colors or the colors stored in the track metadata? 
Please add a comment to the source code.

Reply:
Visual. I'll add a comment.